spec/javascripts/components/story/expanded_story/expanded_story_tooltip_spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryTooltip from 'components/story/ExpandedStory/ExpandedStoryToolTip';
+
+describe('<ExpandedStoryTooltip />', () => {
+  it('renders <Popover />', () => {
```

Comment:
Shouldn't we add more specs? As far as I can see, Okay we have the component rendered, but what about what is being rendered and how it is being rendered?

Reply:
Yeah, but this component uses a `renderContent` prop, which is kinda tricky to test, so we decided to not add much complexity to the test with so small gain.

I see, however, that the coverage dropped a lot, so we're open to suggestions.